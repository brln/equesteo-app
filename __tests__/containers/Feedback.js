import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import FeedbackContainer from '../../containers/Feedback'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    localState: {}
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <FeedbackContainer />
    </Provider>)
  expect(tree.toJSON()).toMatchSnapshot()
})

