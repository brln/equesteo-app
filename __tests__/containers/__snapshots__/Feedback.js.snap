// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutomaticScroll={false}
  enableOnAndroid={false}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  getScrollResponder={[Function]}
  handleOnScroll={[Function]}
  keyboardDismissMode="interactive"
  keyboardOpeningTime={250}
  keyboardSpace={0}
  onScroll={[Function]}
  resetKeyboardSpace={[Function]}
  scrollEventThrottle={1}
  scrollForExtraHeightOnAndroid={[Function]}
  scrollIntoView={[Function]}
  scrollToEnd={[Function]}
  scrollToFocusedInput={[Function]}
  scrollToPosition={[Function]}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "height": 1334,
    }
  }
  update={[Function]}
  viewIsInsideTabBar={false}
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "height": 1394,
          "marginTop": 20,
        }
      }
    >
      <View>
        <TextInput
          allowFontScaling={true}
          maxLength={5000}
          multiline={true}
          onChangeText={[Function]}
          placeholder="Let us know what you think. Likes, dislikes, problems, feature requests, musings, anecdotes, stories, we'd love to hear it.

It sends a message straight to our phones.

We'll email you back at the address you signed up with, let us know if there's a better way to reach you.

Or come to Ben Lomond and ride with us!"
          rejectResponderTermination={true}
          style={
            Object {
              "borderColor": "#4b9ec1",
              "borderWidth": 1,
              "height": 800.4,
              "padding": 10,
              "textAlignVertical": "top",
              "width": 600,
            }
          }
          underlineColorAndroid="transparent"
          value={null}
        />
        <View
          style={
            Object {
              "paddingBottom": 10,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#e8eae8",
                  "width": undefined,
                },
                Object {
                  "alignSelf": "center",
                  "borderRadius": 10,
                  "marginTop": 10,
                  "paddingBottom": 10,
                  "paddingTop": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fff",
                    "paddingLeft": 20,
                    "paddingRight": 20,
                    "textAlign": "center",
                  },
                  Object {
                    "color": "#9b9d9b",
                  },
                ]
              }
            >
              Submit
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
